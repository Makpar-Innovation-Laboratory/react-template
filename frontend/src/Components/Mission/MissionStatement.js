import React from 'react'
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';
import 'bootstrap/dist/css/bootstrap.css';
import MissionCard from './MissionCard'

function MissionStatement() {
    return (
        <div
            className="d-flex flex-column background-light "
            style={{ width: "100%", minHeight: "95vh" }}
        >
           
            <Row xs={2} md={4} lg={6}>
                <Col>
                    <h3>Mission</h3>
                </Col>
                <Col>
                    <h3>Lorem ipsum dolor sit amet</h3>
                </Col>
            </Row>
            <MissionCard></MissionCard>
        </div>
    )
}

export default MissionStatement
