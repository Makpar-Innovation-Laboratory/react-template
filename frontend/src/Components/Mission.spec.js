import React from "react";
import { isolateComponent } from "isolate-react";
import Mission from "./Mission";
// import { shallow, mount } from "enzyme";

describe("JSX Tests :: <Mission /> Rendering", () => {

  test("Sanity Check Snapshot Rendering", () => {
    const wrapper = isolateComponent(<Mission />);

    expect(wrapper.debug()).toMatchSnapshot();
  });

});
