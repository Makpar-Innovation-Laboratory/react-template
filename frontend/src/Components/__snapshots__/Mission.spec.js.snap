// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSX Tests :: <Mission /> Rendering Sanity Check Snapshot Rendering 1`] = `
"{\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-container\\",\\"style\\":{\\"padding\\":\\"3rem 0\\"},\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"display\\":\\"flex\\",\\"justifyContent\\":\\"center\\",\\"width\\":\\"80%\\"},\\"className\\":\\"white-text mission-header\\",\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"width\\":\\"20%\\",\\"textAlign\\":\\"right\\"},\\"children\\":{\\"type\\":\\"h3\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mr-2\\",\\"children\\":\\"MISSION\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"width\\":\\"80%\\",\\"display\\":\\"flex\\",\\"flexDirection\\":\\"column\\",\\"justifyContent\\":\\"flex-start\\"},\\"children\\":[{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title\\",\\"children\\":\\"OUR VALUES\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-header-border\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-subheading-2\\",\\"children\\":\\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Consectetur adipiscing elit, sed do eiusmod tempor incididunt.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"width\\":\\"90%\\",\\"display\\":\\"flex\\",\\"justifyContent\\":\\"space-around\\"},\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"UPLIFT OUR\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\"COMMUNITIES\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize staying engaged with our clients, our colleagues, our work, and our communities.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":\\"ON IT!\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We are enthusiastic about delivering high quality work, making a difference, quick to find solutions to problems, eager to step up to the challenge; in short: a great attitude.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CUSTOMER\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" DELIGHT\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize excellence in our service output, always thinking ahead to anticipate client needs, building trust with our clients overtime through delivering value, and staying on top of making our clients' lives easier.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CALM\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" EXECUTION\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"Under heat, we keep a calm composure â€“ because if we stay calm, our clients and our teams can remain calm. Even when the pressure is on, calm minds can achieve anything they put their minds to. Calm is the magic elixir to our success and harmony.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
 -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"style\\":{\\"display\\":\\"flex\\",\\"justifyContent\\":\\"center\\",\\"width\\":\\"80%\\"},\\"className\\":\\"white-text mission-header\\",\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"width\\":\\"20%\\",\\"textAlign\\":\\"right\\"},\\"children\\":{\\"type\\":\\"h3\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mr-2\\",\\"children\\":\\"MISSION\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"style\\":{\\"width\\":\\"80%\\",\\"display\\":\\"flex\\",\\"flexDirection\\":\\"column\\",\\"justifyContent\\":\\"flex-start\\"},\\"children\\":[{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title\\",\\"children\\":\\"OUR VALUES\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-header-border\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-subheading-2\\",\\"children\\":\\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Consectetur adipiscing elit, sed do eiusmod tempor incididunt.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"style\\":{\\"width\\":\\"20%\\",\\"textAlign\\":\\"right\\"},\\"children\\":{\\"type\\":\\"h3\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mr-2\\",\\"children\\":\\"MISSION\\"},\\"_owner\\":null,\\"_store\\":{}}},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h3\\",\\"name\\":\\"h3\\",\\"props\\":{\\"className\\":\\"mr-2\\",\\"children\\":\\"MISSION\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"MISSION\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"style\\":{\\"width\\":\\"80%\\",\\"display\\":\\"flex\\",\\"flexDirection\\":\\"column\\",\\"justifyContent\\":\\"flex-start\\"},\\"children\\":[{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title\\",\\"children\\":\\"OUR VALUES\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-header-border\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-subheading-2\\",\\"children\\":\\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Consectetur adipiscing elit, sed do eiusmod tempor incididunt.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h5\\",\\"name\\":\\"h5\\",\\"props\\":{\\"className\\":\\"mat-title\\",\\"children\\":\\"OUR VALUES\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"OUR VALUES\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-header-border\\"},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"p\\",\\"name\\":\\"p\\",\\"props\\":{\\"className\\":\\"mat-subheading-2\\",\\"children\\":\\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Consectetur adipiscing elit, sed do eiusmod tempor incididunt.\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam. Consectetur adipiscing elit, sed do eiusmod tempor incididunt.\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
 -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"style\\":{\\"width\\":\\"90%\\",\\"display\\":\\"flex\\",\\"justifyContent\\":\\"space-around\\"},\\"children\\":[{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"UPLIFT OUR\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\"COMMUNITIES\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize staying engaged with our clients, our colleagues, our work, and our communities.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":\\"ON IT!\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We are enthusiastic about delivering high quality work, making a difference, quick to find solutions to problems, eager to step up to the challenge; in short: a great attitude.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CUSTOMER\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" DELIGHT\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize excellence in our service output, always thinking ahead to anticipate client needs, building trust with our clients overtime through delivering value, and staying on top of making our clients' lives easier.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"div\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CALM\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" EXECUTION\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"Under heat, we keep a calm composure â€“ because if we stay calm, our clients and our teams can remain calm. Even when the pressure is on, calm minds can achieve anything they put their minds to. Calm is the magic elixir to our success and harmony.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"UPLIFT OUR\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\"COMMUNITIES\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize staying engaged with our clients, our colleagues, our work, and our communities.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"img\\",\\"name\\":\\"img\\",\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h5\\",\\"name\\":\\"h5\\",\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"UPLIFT OUR\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\"COMMUNITIES\\"]},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"UPLIFT OUR\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
    -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"br\\",\\"name\\":\\"br\\",\\"props\\":{},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"COMMUNITIES\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"p\\",\\"name\\":\\"p\\",\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize staying engaged with our clients, our colleagues, our work, and our communities.\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"We prioritize staying engaged with our clients, our colleagues, our work, and our communities.\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":\\"ON IT!\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We are enthusiastic about delivering high quality work, making a difference, quick to find solutions to problems, eager to step up to the challenge; in short: a great attitude.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"img\\",\\"name\\":\\"img\\",\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h5\\",\\"name\\":\\"h5\\",\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":\\"ON IT!\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"ON IT!\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"p\\",\\"name\\":\\"p\\",\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We are enthusiastic about delivering high quality work, making a difference, quick to find solutions to problems, eager to step up to the challenge; in short: a great attitude.\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"We are enthusiastic about delivering high quality work, making a difference, quick to find solutions to problems, eager to step up to the challenge; in short: a great attitude.\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CUSTOMER\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" DELIGHT\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize excellence in our service output, always thinking ahead to anticipate client needs, building trust with our clients overtime through delivering value, and staying on top of making our clients' lives easier.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"img\\",\\"name\\":\\"img\\",\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h5\\",\\"name\\":\\"h5\\",\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CUSTOMER\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" DELIGHT\\"]},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"CUSTOMER\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
    -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"br\\",\\"name\\":\\"br\\",\\"props\\":{},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\" DELIGHT\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"p\\",\\"name\\":\\"p\\",\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"We prioritize excellence in our service output, always thinking ahead to anticipate client needs, building trust with our clients overtime through delivering value, and staying on top of making our clients' lives easier.\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"We prioritize excellence in our service output, always thinking ahead to anticipate client needs, building trust with our clients overtime through delivering value, and staying on top of making our clients' lives easier.\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
  -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"div\\",\\"name\\":\\"div\\",\\"props\\":{\\"className\\":\\"mission-card primary-text\\",\\"children\\":[{\\"type\\":\\"img\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"h5\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CALM\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" EXECUTION\\"]},\\"_owner\\":null,\\"_store\\":{}},{\\"type\\":\\"p\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"Under heat, we keep a calm composure â€“ because if we stay calm, our clients and our teams can remain calm. Even when the pressure is on, calm minds can achieve anything they put their minds to. Calm is the magic elixir to our success and harmony.\\"},\\"_owner\\":null,\\"_store\\":{}}]},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"img\\",\\"name\\":\\"img\\",\\"props\\":{\\"src\\":{},\\"height\\":\\"180px\\"},\\"key\\":\\"null\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"h5\\",\\"name\\":\\"h5\\",\\"props\\":{\\"className\\":\\"mat-headline text-primary\\",\\"children\\":[\\"CALM\\",{\\"type\\":\\"br\\",\\"key\\":null,\\"ref\\":null,\\"props\\":{},\\"_owner\\":null,\\"_store\\":{}},\\" EXECUTION\\"]},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"CALM\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
    -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"br\\",\\"name\\":\\"br\\",\\"props\\":{},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\" EXECUTION\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}
   -> {\\"nodeType\\":\\"html\\",\\"type\\":\\"p\\",\\"name\\":\\"p\\",\\"props\\":{\\"className\\":\\"mat-title text-primary\\",\\"children\\":\\"Under heat, we keep a calm composure â€“ because if we stay calm, our clients and our teams can remain calm. Even when the pressure is on, calm minds can achieve anything they put their minds to. Calm is the magic elixir to our success and harmony.\\"},\\"key\\":\\"null\\"}
    -> {\\"nodeType\\":\\"string\\",\\"type\\":\\"Under heat, we keep a calm composure â€“ because if we stay calm, our clients and our teams can remain calm. Even when the pressure is on, calm minds can achieve anything they put their minds to. Calm is the magic elixir to our success and harmony.\\",\\"props\\":{},\\"name\\":\\"\\",\\"key\\":\\"undefined\\"}"
`;
